woocommerce/woocommerce-ios
# Lightweight Storefront: Themes option in Settings

Closes: #11303, #11304


## Description

This PR:
- Adds feature flag for lightweight storefront,
- Adds a use case for checking lightweight storefront eligiblity,
- Adds a "Themes" setting in Settings > Store Settings area for eligible sites.

The "Themes" setting will not do anything yet, waiting for the themes selection flow to be built first.

## Testing instructions


1. Use a WordPress.com site (can be regular Atomic site or Woo Express)
2. Go to Settings, ensure "Themes" is listed in "Store Settings".

## Screenshots


<img src="https://github.com/woocommerce/woocommerce-ios/assets/266376/e5a1d71c-2b96-4898-baf2-1f488d6c7255" width="42%" />
